scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -17,6 +18,18 @@ public class PrepareTargets
         [Target(nameof(Init), nameof(RestorePackages))]
         public static BuildTargetResult Prepare(BuildTargetContext c) => c.Success();
 
+        [Target(nameof(CheckPrereqCmakePresent), nameof(CheckPlatformDependencies))]
+        public static BuildTargetResult CheckPrereqs(BuildTargetContext c) => c.Success();
+
+        [Target(nameof(CheckCoreclrPlatformDependencies),  nameof(CheckInstallerBuildPlatformDependencies))]
+        public static BuildTargetResult CheckPlatformDependencies(BuildTargetContext c) => c.Success();
+
+        [Target(nameof(CheckUbuntuCoreclrDependencies),  nameof(CheckCentOSCoreclrDependencies))]
+        public static BuildTargetResult CheckCoreclrPlatformDependencies(BuildTargetContext c) => c.Success();
```

Comment:
This is going to require further investigation. I think it should be out of scope for this PR and I've assigned myself an issue here: https://github.com/dotnet/cli/issues/1508